evap/staff/urls.py

```diff
@@ -84,6 +84,8 @@
     path("faq/", views.faq_index, name="faq_index"),
     path("faq/<int:section_id>", views.faq_section, name="faq_section"),
 
+    path("edit_infotexts/", views.edit_infotexts, name="edit_infotexts"),
```

Comment:
regarding naming: I think just "infotexts" as the url and `views.infotexts` or maybe `views.infotext_edit` would be more consistent with what we have so far:
* Views are usually named `course_edit`, not `edit_course`.
* The other "smaller" staff pages, e.g. FAQ or text answer warnings, are also "edit" views, but don't necessarily have that in their name.